packages/expo-video/ios/VideoView.swift

```diff
@@ -136,10 +161,47 @@ public final class VideoView: ExpoView, AVPlayerViewControllerDelegate {
   }
 
   public func playerViewControllerDidStartPictureInPicture(_ playerViewController: AVPlayerViewController) {
+    isInPictureInPicture = true
     onPictureInPictureStart()
   }
 
   public func playerViewControllerDidStopPictureInPicture(_ playerViewController: AVPlayerViewController) {
+    isInPictureInPicture = false
     onPictureInPictureStop()
   }
+
+  // MARK: - Utils
+
+  /**
+   * iOS automatically pauses videos when the app enters the background. Only way of avoiding this is to detach the player from the playerLayer.
+   * Typical way of doing this for `AVPlayerViewController` is setting `playerViewController.player = nil`, but that makes the
+   * video invisible for around a second after foregrounding, disabling the tracks requires more code, but works a lot faster.
+   */
+  private func setPlayerTracksEnabled(enabled: Bool) {
+    if let player, let tracks = player.currentItem?.tracks {
+      tracks.forEach { track in
```

Comment:
If it's possible, try to avoid unnecessary indentation. You can do `player?.currentItem?.tracks?.forEach` or just `guard let tracks = player?.currentItem?.tracks else { return }`.